packages/react-components/react-infolabel/library/src/components/InfoButton/useInfoButtonStyles.styles.ts

```diff
@@ -95,6 +95,9 @@ const useButtonStyles = makeStyles({
 });
 
 const usePopoverSurfaceStyles = makeStyles({
+  base: {
+    maxWidth: '240px',
```

Comment:
@khmakoto 👋🏻

Question: why it should have the same max-width as Tooltip? According to Figma designs, the Popover for InfoButton has 264px width. If we decide to limit the max-width and it should match Tooltip width, then it probably have to be unified

Reply:
Hey @mainframev, you are right, I've updated the PR and its description to be `264px` instead.